Source/control.cpp

```diff
@@ -383,8 +378,32 @@ void RemoveGold(Player &player, int goldIndex)
 	dropGoldValue = 0;
 }
 
+bool IsLevelUpButtonVisible()
+{
+	if (spselflag || chrflag || Players[MyPlayerId]._pStatPts == 0) {
+		return false;
+	}
+	if (ControlMode == ControlTypes::VirtualGamepad) {
+		return false;
+	}
+	if (stextflag != STORE_NONE || IsStashOpen) {
+		return false;
+	}
+	if (QuestLogIsOpen && GetLeftPanel().Contains(GetMainPanel().position + Displacement { 0, -74 })) {
+		return false;
+	}

```

Comment:
In this case, does it make more sense to treat stash the same as store text or the quest log?

Reply:
Since it is accessed via the store and you can't move with it open I went with treating it more as a store. I would also like to use it as a basis for an eventual graphical store.